README.md

```diff
@@ -253,6 +254,76 @@ https://youtu.be/dQw4w9WgXcQ
 ></iframe>
 ```
 
+### Custom Transformers
```

Comment:
Maybe also make this header at the same level (`h2`) as our built-in transformers header, so it's a bit clearer that this is something else? 🤔

Don't forget to check all sub-headers too

Reply:
Sure thing!